ocaml/testsuite/tests/typing-layouts-err-msg/concrete_v1_check.compilers.reference

```diff
@@ -0,0 +1,9 @@
+type t_void : void
+File "_none_", line 1:
+Error: Non-value detected in [translmod]:
+       Please report this error to the Jane Street compilers team.
+       The layout of t_void is void, because
+         it's part of the v1 safety check, defaulted to layout void.
```

Comment:
I'm a little alarmed at "defaulted". We should never be defaulting to void anymore. (We did for a short time about a year ago.)

Reply:
See the comment on `concrete_v1_check.ml`